pipeline/data_preprocessing/genotype/VCF_QC.ipynb

```diff
@@ -174,9 +178,14 @@
     "parameter: job_size = 1\n",
     "# Walltime \n",
     "parameter: walltime = '5h'\n",
-    "parameter: mem = '16G'\n",
+    "parameter: mem = '60G'\n",
```

Comment:
Why do we need 60GB memory by default?

Reply:
In both step 16GB is not enough and in diana's notebook she use 60Gb

Reply:
> In both step 16GB is not enough

Not enough for our ROSMAP data? 60GB sounds too much to me. Diana's UKB application requires more memory for a huge number of samples. We are under 10K samples almost surely. I just want to decide on a reasonable default here. ,...

Reply:
16Gb will give an error of not enough memory...I havnt tested the others though.

btw all other things was fixed